ansible-playbooks/roles/connect-aks-to-acm-hub-collection/tasks/delete.yml

```diff
@@ -25,7 +48,8 @@
 
 - name: Get network interfaces within a resource group
   azure.azcollection.azure_rm_networkinterface_info:
-    resource_group: "{{ AKS_MRG }}"
+    resource_group: "{{ AKS_VNET_RG
```

Comment:
typo here ?

Reply:
Clear... But I tested multiple time and it worked... Please don't merge

Reply:
Ah ok. 
`- hosts: localhost
  vars:
     myvar_es: "mundo"
     myvar_us: "world"
  connection: local
  gather_facts: no
  become: false
  tasks:
  - name: this is my test
    debug:
      msg: "hello {{ myvar_es
}} "
`

results in:
`$ ansible-playbook  play.yml 
[WARNING]: provided hosts list is empty, only localhost is available. Note that the implicit localhost does not match 'all'

PLAY [localhost] **********************************************************************************************************************************************************************************************************************************************************************************************************

TASK [this is my test] ****************************************************************************************************************************************************************************************************************************************************************************************************
ok: [localhost] => {
    "msg": "hello mundo "
}

PLAY RECAP ****************************************************************************************************************************************************************************************************************************************************************************************************************
localhost                  : ok=1    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0   

`

Reply:
it looks like the `\n` is skipped inside a `jinjavar`

Reply:
:) finally it's a nit :) But I fix it anyway